templates/marathon.tmpl

```diff
@@ -1,13 +1,13 @@
 {{$apps := .Applications}}
 [backends]{{range .Tasks}}
-    [backends.backend{{.AppID | replace "/" "-"}}.servers.server-{{.ID | replace "." "-"}}]
+    [backends.backend{{getBackend .}}.servers.server-{{.ID | replace "." "-"}}]
```

Comment:
breaks because this is inside the range for .Tasks so its passing a task instead of an application. so we need to pass back through the application here, but i'm not sure on how to do that